index.bs

```diff
@@ -1341,18 +1345,6 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Assert: <code>|options|.{{CredentialCreationOptions/publicKey}}</code> is [=present=].
 
-1. If <var ignore>sameOriginWithAncestors</var> is [FALSE], return a "{{NotAllowedError}}" {{DOMException}}.
```

Comment:
The equivalent of this check will be moved to CredMan instead, right?

Reply:
yes. See https://github.com/w3c/webappsec-credential-management/pull/138